### PR TITLE
Additional tests for the project member endpoint

### PR DESCRIPTION
**Pull request by alexeyleping**
add new tests and rename bad names

### CODE REVIEW

**Comment by alexeyleping** [comment]
@astanik I don't really understand why the tests fail if everything is going locally
Screenshot from 2024-07-05 09-18-32
Screenshot from 2024-07-05 09-19-00

**Comment by alexeyleping** [comment]
I don't really understand why the tests fail if every thing is going locally
Screenshot from 2024-07-05 09-18-32
Screenshot from 2024-07-05 09-19-00

**Comment by alexeyleping** [comment]
@astanik When merging branches, you have duplicated the methods

File: remsfal-service/src/test/java/de/remsfal/service/boundary/ProjectResourceTest.java [hunk 1]
```diff
@@ -32,60 +30,60 @@ protected void setupTestUsers() {
     @Test
     void getProjects_FAILED_noAuthentication() {
         given()
-            .when()
-            .get(BASE_PATH)
-            .then()
-            .statusCode(Status.UNAUTHORIZED.getStatusCode());
+                .when()
```
Comment (astanik) [inline remsfal-service/src/test/java/de/remsfal/service/boundary/ProjectResourceTest.java:33]: Please remove the unnecessary indentation. I added a rule that will check this in the future. As you can see in this PR https://github.com/remsfal/remsfal-backend/pull/180

We will also have a formatter which is currently in work in this PR https://github.com/remsfal/remsfal-backend/pull/166

**Review by astanik — COMMENTED** [review]
Many unnecessary changes make the code difficult to review.

Sonar coverage analysis can be ignored because unchanged code is not covered enough.

**Comment by astanik** [comment]
@alexeyleping I have no idea why you got duplicate methods after merge! I started a review in order to close this issue. If you like this project, we can also collaborate in additional issues?

**Comment by alexeyleping** [comment]
@astanik I have removed the margins, and I am ready to cooperate in the future!

**Pull request merged**